server/src/main/kotlin/org/jetbrains/bsp/bazel/server/sync/BspProjectMapper.kt

```diff
@@ -68,23 +75,25 @@ import org.jetbrains.bsp.bazel.server.sync.languages.jvm.javaModule
 import org.jetbrains.bsp.bazel.server.sync.languages.scala.ScalaModule
 import org.jetbrains.bsp.bazel.server.sync.model.Label
 import org.jetbrains.bsp.bazel.server.sync.model.Language
+import org.jetbrains.bsp.bazel.server.sync.model.Library
 import org.jetbrains.bsp.bazel.server.sync.model.Module
 import org.jetbrains.bsp.bazel.server.sync.model.Project
 import org.jetbrains.bsp.bazel.server.sync.model.Tag
 import org.jetbrains.bsp.bazel.workspacecontext.WorkspaceContextProvider
 import java.net.URI
 import java.nio.file.Path
 import java.nio.file.Paths
+import java.util.*
 import kotlin.io.path.name
 import kotlin.io.path.relativeToOrNull
 import kotlin.io.path.toPath
 
 class BspProjectMapper(
-  private val languagePluginsService: LanguagePluginsService,
-  private val workspaceContextProvider: WorkspaceContextProvider,
-  private val bazelPathsResolver: BazelPathsResolver,
-  private val bazelRunner: BazelRunner,
-  private val bspInfo: BspInfo,
+    private val languagePluginsService: LanguagePluginsService,
```

Comment:
where do these formatting changes come from?

Reply:
Fixed, Intellij didn't like the 8 space indents.